segtypes/gc/relheader.py

```diff
@@ -0,0 +1,112 @@
+from segtypes.common.header import CommonSegHeader
+from util import options
+
+
+class RelSegHeader(CommonSegHeader):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.version: int = self.yaml.get("version", 0)
```

Comment:
This usually has an `isinstance` check first instead of using the yaml directly
```suggestion
        if isinstance(self.yaml, dict):
            self.version: int = self.yaml.get("version", 0)
```